include/librfu.h

```diff
@@ -1,58 +1,281 @@
 #ifndef GUARD_LIBRFU_H
 #define GUARD_LIBRFU_H
 
+#include "global.h"
 #include "main.h"
 
-enum
-{
-    RFU_RESET = 0x10,
-    RFU_LINK_STATUS,
-    RFU_VERSION_STATUS,
-    RFU_SYSTEM_STATUS,
-    RFU_SLOT_STATUS,
-    RFU_CONFIG_STATUS,
-    RFU_GAME_CONFIG,
-    RFU_SYSTEM_CONFIG,
-    RFU_UNK18,
-    RFU_SC_START,
-    RFU_SC_POLLING,
-    RFU_SC_END,
-    RFU_SP_START,
-    RFU_SP_POLLING,
-    RFU_SP_END,
-    RFU_CP_START,
-    RFU_CP_POLLING,
-    RFU_CP_END,
-    RFU_UNK22,
-    RFU_UNK23,
-    RFU_DATA_TX,
-    RFU_DATA_TX_AND_CHANGE,
-    RFU_DATA_RX,
-    RFU_MS_CHANGE,
-    RFU_DATA_READY_AND_CHANGE,
-    RFU_DISCONNECTED_AND_CHANGE,
-    RFU_UNK2A,
-    RFU_UNK2B,
-    RFU_UNK2C,
-    RFU_UNK2D,
-    RFU_UNK2E,
-    RFU_UNK2F,
-    RFU_DISCONNECT,
-    RFU_TEST_MODE,
-    RFU_CPR_START,
-    RFU_CPR_POLLING,
-    RFU_CPR_END,
-    RFU_UNK35,
-    RFU_UNK36,
-    RFU_RESUME_RETRANSMIT_AND_CHANGE,
-    RFU_UNK38,
-    RFU_UNK39,
-    RFU_UNK3A,
-    RFU_UNK3B,
-    RFU_UNK3C,
-    RFU_STOP_MODE, // 3D
-};
+/* TODOs: 
+ * - split files
```

Comment:
Are these TODOs still valid?

Reply:
I deleted the first one. Other ones should still be there